staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/config.go

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Can you please split this PR into two separate PRs:
- the first one introducing this "WorkEstimatorConfig" structure and piping that through to estimators, but still using the default values everywhere
- the second PR that is exposing the functionality to pass a config file to kube-apiserver

This will make the code change smaller and split the two things:
- the first PR is effectively a non-controversial refactoring
- the second is exposing a new option, so is effectively a change on how kube-apiserver can be used

Reply:
Done. Edited this PR to only contain changes related to the `Config` structures and moved the other changes to https://github.com/kubernetes/kubernetes/pull/111222.